core/server/master/src/test/java/alluxio/master/meta/AlluxioMasterRestServiceHandlerTest.java

```diff
@@ -431,4 +441,108 @@ public void getWorkerInfoList() {
+      assertNotNull("Response must have a entry!", response.getEntity());
+      assertEquals("Entry must be a MasterWebUIInit!", MasterWebUIInit.class,
+          response.getEntity().getClass());
+    } finally {
+      response.close();
+    }
+  }
+
+  @Test
+  public void getWebUIOverview() {
+    Response response = mHandler.getWebUIOverview();
+    try {
+      assertNotNull("Response must be not null!", response);
+      assertNotNull("Response must have a entry!", response.getEntity());
+      assertEquals("Entry must be a MasterWebUIOverview!", MasterWebUIOverview.class,
+          response.getEntity().getClass());
+    } finally {
+      response.close();
+    }
+  }
+
+  @Test
+  public void getWebUIBrowse() {
+    Response response = mHandler.getWebUIBrowse("test", "0", "", "20");
+    try {
+      assertNotNull("Response must be not null!", response);
+      assertNotNull("Response must have a entry!", response.getEntity());
+      assertEquals("Entry must be a MasterWebUIBrowse!", MasterWebUIBrowse.class,
+          response.getEntity().getClass());
+    } finally {
+      response.close();
+    }
+  }
+
+  @Test
+  public void getWebUIData() {
+    Response response = mHandler.getWebUIData("0", "20");
+    try {
+      assertNotNull("Response must be not null!", response);
+      assertNotNull("Response must have a entry!", response.getEntity());
+      assertEquals("Entry must be a MasterWebUIData!", MasterWebUIData.class,
+          response.getEntity().getClass());
+    } finally {
+      response.close();
+    }
+  }
+
+  @Test
+  public void getWebUILogs() {
+    Response response = mHandler.getWebUILogs("", "0", "", "20");
+    try {
+      assertNotNull("Response must be not null!", response);
+      assertNotNull("Response must have a entry!", response.getEntity());
+      assertEquals("Entry must be a MasterWebUILogs!", MasterWebUILogs.class,
+          response.getEntity().getClass());
+    } finally {
+      response.close();
+    }
+  }
+
+  @Test
+  public void getWebUIConfiguration() {
+    Response response = mHandler.getWebUIConfiguration();
+    try {
+      assertNotNull("Response must be not null!", response);
+      assertNotNull("Response must have a entry!", response.getEntity());
+      assertEquals("Entry must be a MasterWebUIConfiguration!", MasterWebUIConfiguration.class,
+          response.getEntity().getClass());
+    } finally {
+      response.close();
+    }
+  }
+
+  @Test
+  public void getWebUIWorkers() {
+    Response response = mHandler.getWebUIWorkers();
+    try {
+      assertNotNull("Response must be not null!", response);
+      assertNotNull("Response must have a entry!", response.getEntity());
+      assertEquals("Entry must be a MasterWebUIWorkers!", MasterWebUIWorkers.class,
+          response.getEntity().getClass());
+    } finally {
+      response.close();
+    }
+  }
+
+  @Test
+  public void getWebUIMetrics() {
+    Response response = mHandler.getWebUIMetrics();
+    try {
```

Comment:
this pattern is repeated quite a few times, could we have a helper function that takes a response and expected entity type?

Reply:
this will probably be a //TODO